irange for PyTorch sans jit

r-barnes opened the pull request:
Differential Revision: D28909681

cbalioglu commented:
CI is mostly green. Since the PR is fairly large, I just skimmed through it. Besides introducing range based loops are there any behavioral changes? And just out of curiosity; did you use some sed/awk magic or did you make the changes manually?

r-barnes committed:
irange for PyTorch sans jit (#59481)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/59481

Differential Revision: D28909681

fbshipit-source-id: a189f103a06ed2f5078dd65e28bdba8cf9d23405
